approval_frame/templates/registration/password_reset_form.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <div class='row-fluid top-buffer'>
+      <div class='col-md-6 col-md-offset-3'>
+        <legend>Reset Password</legend>
+        <p>Enter your email address below and we will send you an email with
+         instructions to reset your password.</p>
+        <form action="." method="post">
+        {% csrf_token %}
+          <div class='form-group {% if form.errors.email %} has-error {% endif %}'>
+            {% for error in form.errors.email %}
+              <label class='control-label' for='{{ form.email.id_for_label }}'>
+                {{ error }}
+              </label>
+            {% endfor %}
+            <input type='text' class='form-control' id='{{ form.email.id_for_label }}'
+                  name='{{ form.email.name }}' placeholder='Email ID'
+                  maxlength={{ form.email.field.max_length }}>
+          </div>
+          <button type='submit' class='btn btn-success'>Submit</button>
```

Comment:
I was attempting to conceal the fact that a link would be sent to the email address (for security purposes) by specifying "instructions" in the prompt. If that is not required, I could change both the prompt to "A link to reset your password will be sent to the email address entered below." and subsequently the button "Send reset link"